x/upgrade/abci.go

```diff
@@ -20,10 +22,15 @@ import (
 // The purpose is to ensure the binary is switched EXACTLY at the desired block, and to allow
 // a migration to be executed if needed upon this switch (migration defined in the new binary)
 // skipUpgradeHeightArray is a set of block heights for which the upgrade must be skipped
-func BeginBlocker(k *keeper.Keeper, ctx sdk.Context) {
+func BeginBlocker(ctx context.Context, k *keeper.Keeper) error {
```

Comment:
API breaking changelog is needed for that in upgrade.

Reply:
I've added all of these into the module's own changelog
https://github.com/cosmos/cosmos-sdk/pull/16227/files#diff-10b3113dc02d20007dd57d9a631f39c407015d7c765f8c060646c05dfbb8f0ffR32-R35